src/screens/GroupSetting/elements/EditGroup.js

```diff
@@ -107,23 +107,23 @@ const styles = StyleSheet.create({
     marginBottom: 5
   },
   textDesc: {
-    color: COLORS.white,
+    color: colors.white,
     fontFamily: fonts.inter[400],
     fontSize: normalizeFontSize(12),
     lineHeight: normalizeFontSize(18)
   },
   container: {
-    backgroundColor: COLORS.holyTosca,
+    backgroundColor: colors.darkBlue,
     borderTopWidth: 1,
-    borderTopColor: COLORS.alto,
+    borderTopColor: colors.alto,
     paddingVertical: 10,
     paddingHorizontal: 15
   },
   btnUpdatePhoto: {
     width: normalize(48),
     height: normalize(48),
     borderRadius: normalize(24),
-    backgroundColor: COLORS.alto,
+    backgroundColor: colors.alto,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here seem to be related to styling. However, it's important to ensure that the new colors being used (`colors.white`, `colors.darkBlue`, `colors.alto`) are defined in your color utility file and they match with the theme of your application. If not, you might want to reconsider the color choices or add them to your color utility file.


<!-- This is an auto-generated comment by OSS CodeRabbit -->